spec-main/spec-helpers.ts

```diff
@@ -2,9 +2,19 @@ import * as childProcess from 'child_process';
 import * as path from 'path';
 import * as http from 'http';
 import * as v8 from 'v8';
+import { SuiteFunction, TestFunction } from 'mocha';
```

Comment:
This stuff I just added because it was annoying me, you can now do `ifdescribe().only`